### PR TITLE
make workflows match template

### PR DESCRIPTION
**Pull request by jukent**
https://github.com/ProjectPythiaCookbooks/projectpythiacookbooks.github.io/issues/96#issuecomment-1308765962

### CODE REVIEW

File: .github/workflows/nightly-build.yaml [hunk 1]
```diff
@@ -7,15 +7,12 @@ on:
 
 jobs:
   build:
+    if: ${{ github.repository_owner == 'ProjectPythiaCookbooks' }}
     uses: ProjectPythiaCookbooks/cookbook-actions/.github/workflows/build-book.yaml@main
     with:
       environment_name: cookbook-dev
```
Comment (brian-rose) [inline .github/workflows/nightly-build.yaml:13]:
```suggestion
      environment_name: intake-cookbook-dev
```

File: .github/workflows/publish-book.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build:
+    uses: ProjectPythiaCookbooks/cookbook-actions/.github/workflows/build-book.yaml@main
+    with:
+      environment_name: cookbook-dev
```
Comment (brian-rose) [inline .github/workflows/publish-book.yaml:14]:
```suggestion
      environment_name: intake-cookbook-dev
```

File: .github/workflows/trigger-book-build.yaml [hunk 3]
```diff
@@ -7,13 +7,5 @@ jobs:
     uses: ProjectPythiaCookbooks/cookbook-actions/.github/workflows/build-book.yaml@main
     with:
       environment_name: cookbook-dev
```
Comment (brian-rose) [inline .github/workflows/trigger-book-build.yaml:9]:
```suggestion
      environment_name: intake-cookbook-dev
```

**Review by brian-rose — CHANGES_REQUESTED** [review]
@jukent this all looks good except the environment name that gets passed to the build-book action in several places. I think this passed tests only because it was using a cached version of the build environment.

**Comment by jukent** [comment]
I'll look for that in the other PRs too

**Comment by brian-rose** [comment]
> @brian-rose in the other cookbooks, the environment name is just "cookbook-dev" so nothing else needs to be changed in the workflows. Is there a reason to give it a repo specific name?

I think it's a decision for individual cookbook authors to make. The reason is to disambiguate that environment from any number of other cookbook environments that an individual might have installed locally.

**Review by brian-rose — APPROVED** [review]

**Comment by jukent** [comment]
Makes sense.

**Pull request merged**